package.json

```diff
@@ -28,7 +28,7 @@
     "build": "npm run clean && npm run build-css && npm run build-js",
     "build-production": "export NODE_ENV=production && npm run build && ./bin/hash-rev",
     "build-autotrack": "npx autotrack -o public/autotrack/autotrack.js -p outboundLinkTracker,eventTracker",
-    "watch-css": "watch 'npm run build-css' assets/sass",
+    "watch-css": "watch 'npm run build-css' assets/sass --interval=0.1",
```

Comment:
?

Reply:
I was seeing a really slow (like, 2-3 seconds) delay on seeing file changes here. [This](https://github.com/mikeal/watch/pull/84#issuecomment-193960026) helped a bit, so adding it here while I remembered.

Reply:
Oh wow didn't realise the default was 5 seconds! (according to that link), go for it